commons/src/test/java/io/hyscale/commons/utils/HyscaleFilesUtilTests.java

```diff
@@ -105,7 +104,7 @@ public void updateFileTest() throws HyscaleException {
     @Test
     public void copyFileToDirTest() throws HyscaleException {
         HyscaleFilesUtil.copyFileToDir(sampleFile, testDir);
-        assertTrue(new File(testDirPath,sampleFileName).exists());
+        assertTrue(new File(testDirPath, SAMPLE_FILE_NAME).exists());
         assertTrue(testDir.isDirectory());
         assertTrue(testDir.exists());
```

Comment:
Can you check the order of Assertions
 The directory should exist & then check for the file inside the directory

Reply:
did

Reply:
isDirectory checks ifExists internally, you can remove that check